hack/generate-integration-compatibility-versions-test.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
```

Comment:
On reflection: maybe we should put this script in test/integration/apiserver?

Having a script scoped to a single file, not part of automated updates/verify, but also off in the repo-wide hack folder seems like it's just making hack harder to dig through and the script harder to find when working on the tests.

Reply:
+1

Reply:
+1, I've moved this script now to ` test/integration/apiserver/generate-integration-compatibility-versions-test.sh`